src/main.c

```diff
@@ -109,6 +109,7 @@ int     nlch = '\n';
 
 bool    tablesext, tablesverify, gentables;
 char   *tablesfilename=0,*tablesname=0;
+bool    is_tablesfilename_alloc;
```

Comment:
@whydoubt 
Could you please point to me in which part of the code can `tablesfilename` point to a non-allocated buffer.